common_markdown_files/CONTRIBUTING.md.erb

```diff
@@ -0,0 +1,20 @@
+# Contributing
+
+RSpec is a community-driven project that has benefited from improvements from over over *500* contributors.
+We welcome contributions from *everyone*. While contributing, please follow the project [code of conduct](../code_of_conduct.md), so that everyone can be included.
```

Comment:
Hmm, this code of conduct link doesn't appear to be working (at least not from https://github.com/rspec/rspec-core/blob/update-common-markdown-files-2015-07-24-for-master/CONTRIBUTING.md).

Any one got an idea for how to fix this?  There's also the DEVELOPMENT file link below with the same problem.